src/components/pages/ConfirmResetPassword/index.tsx

```diff
@@ -12,8 +12,8 @@ import AuthSuccessMessage, { imageTypes } from 'components/ui/AuthMessage';
 import Link from 'components/ui/Link';
 import { changeUserPassword } from 'store/actions/auth';
 import LocationHelper from 'helpers/Location';
-import DateHelper from 'helpers/DateHelper';
 import SEO from 'components/HOC/SEO';
+import { convertISOtoDateObj } from 'helpers/dateHelpers';
```

Comment:
These are becoming more confusing by the minute.
The `utils/misc` file is supposed to be used to store utility functions used on the front-end for little transformation e.t.c
Helpers are mostly `JSX` like component that performs little functionalities which can be classified as HOCs.